styles.css

```diff
@@ -162,293 +132,559 @@ a:active {
+  }
+
+  .sec_third {
+    width: 98vw;
+    display: flex;
+    flex-direction: row;
+  }
+
+  #card2 {
+    height: 350px;
+    width: 65%;
+    background-image: url(img/image-geometry_circle\ and\ dot.svg);
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  #card2 img {
+    margin-top: 80px;
+    width: 47.5%;
+  }
+
+  #card3 {
+    margin-left: auto;
+    width: 35%;
+  }
+
+  #card3 img {
+    margin-left: auto;
+    width: 80%;
+  }
+
+  #card3 .workdetails {
+    width: 98.9%;
+    height: 195px;
+    margin-top: -250px;
+  }
+
+  .four_five {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 0;
+    height: 55.2vh;
+  }
+
+  #card4 {
+    margin-top: -20px;
+    width: 50%;
+    background-image: url(img/image-geometry_merge.svg);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  #card4 img {
+    width: 84.9%;
+    margin-top: -74px;
+    margin-left: auto;
+  }
+
+  #card5 {
+    width: 69%;
+  }
+
+  #card5 img {
+    height: auto;
+  }
+
+  .lastcard {
+    width: 87.5vw;
+    display: flex;
+  }
+
+  #card2 .workdetails {
+    display: none;
+  }
+
+  #card4 .workdetails {
+    display: none;
+  }
+
+  #card5 .workdetails {
+    display: none;
+  }
+
+  #card6 .workdetails {
+    display: none;
+  }
+
+  .lastbg {
+    width: 95%;
+    background:
+      url(img/image-geometry_5.svg) top left no-repeat,
+      url(img/image-geometry_yellow.svg) top right no-repeat;
+  }
+
+  #card6 {
+    width: 50%;
+  }
+
+  .midimage {
+    display: grid;
+    height: 47vh;
+    background-size: 100% 100%;
+  }
+
+  #about-section {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    margin-right: 100px;
+    margin-top: 210px;
+    margin-bottom: 150px;
+  }
+
+  .about-me {
+    width: 40%;
+  }
+
+  .boxinfo {
+    margin: 0;
+  }
+
+  .wrapbox {
+    margin-left: 100px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .boximage {
+    background-image: url(img/image_dots_with_teal.svg);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .footer-sec {
+    padding-left: 100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .social_icons {
+    margin: 0 160px;
+    width: 30%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .footer-head {
+    display: none;
+  }
+
+  .formbody .cv-btn {
+    align-self: center;
+    text-align: center;
+  }
+
+  .forbackground {
+    background: url(img/image-geometry_shapes1.svg) left top no-repeat, url(img/image-geometry_shapes2.svg) right bottom no-repeat;
+    background-color: #f9f9fa;
+  }
+
+  .forbackground .formbody {
+    margin: auto;
+    text-align: justify;
+    width: 35vw;
+  }
+
+  .name_and_email {
+    width: 35vw;
+    display: flex;
+  }
```

Comment:
- [ ] Please take note that you need to add margin in-between the input boxes according to the design template. As developers, when we pay attention to details; it shows that we can deliver as expected.😃 

## Your Design

![b9](https://user-images.githubusercontent.com/103386183/199996272-38aa77ba-65a9-4f07-9876-0f9546afff61.png)

## Figma Design

![b7](https://user-images.githubusercontent.com/103386183/199996021-40b5d1cb-7a2a-4782-b558-3f6eb2e23fc3.png)